activities/communication/using-chat.md

```diff
@@ -0,0 +1,54 @@
+---
+type: guide
+---
+
+# How to use [chat.publiccode.net](https://chat.publiccode.net/)
```

Comment:
There is no `explains` here and I think the title could be snappier. In addition to that this guide is clearly only for staff, that seems wise to add to the title.

```suggestion
type: guide
explains: What our staff chat tool is, how to get started and it's do's and don'ts
---

# Using [chat.publiccode.net](https://chat.publiccode.net/) for staff
```